apps/prairielearn/src/pages/instructorAssessmentManualGrading/assessmentQuestion/assessmentQuestion.html.tsx

```diff
@@ -144,230 +114,42 @@ export function AssessmentQuestion({
-                          })}
-                        >
-                          <i class="fas fa-user-tag" />
-                          Assign to: {grader.name || ''} ({grader.uid})
-                        </button>
-                      ))}
-                      <button
-                        class="dropdown-item"
-                        type="submit"
-                        name="batch_action_data"
-                        value={JSON.stringify({ assigned_grader: null })}
-                      >
-                        <i class="fas fa-user-slash" />
-                        Remove grader assignment
-                      </button>
-                      <div class="dropdown-divider" />
-                      <button
-                        class="dropdown-item"
-                        type="submit"
-                        name="batch_action_data"
-                        value={JSON.stringify({ requires_manual_grading: true })}
-                      >
-                        <i class="fas fa-tag" />
-                        Tag as required grading
-                      </button>
-                      <button
-                        class="dropdown-item"
-                        type="submit"
-                        name="batch_action_data"
-                        value={JSON.stringify({ requires_manual_grading: false })}
-                      >
-                        <i class="fas fa-check-square" />
-                        Tag as graded
-                      </button>
-                    </div>
-                  </div>
-                )}
-              </div>
-            </div>
-            <table id="grading-table" aria-label="Instance questions for manual grading" />
-          </div>
-          <div
-            // eslint-disable-next-line @eslint-react/dom/no-dangerously-set-innerhtml
-            dangerouslySetInnerHTML={{
-              __html: GroupInfoModal({
-                modalFor: 'selected',
-                numOpenInstances: num_open_instances,
-                csrfToken: __csrf_token,
-              }).toString(),
-            }}
+        <Hydrate fullHeight>
+          <AssessmentQuestionManualGrading
+            authzData={authz_data}
+            search={search}
+            instanceQuestions={instanceQuestions}
+            course={course}
+            courseInstance={course_instance}
+            urlPrefix={urlPrefix}
+            csrfToken={__csrf_token}
+            assessmentId={assessment.id}
+            assessmentQuestionId={assessment_question.id}
+            assessmentQuestion={assessment_question as AssessmentQuestion}
+            assessmentTid={assessment.tid}
+            questionQid={question.qid}
+            aiGradingMode={aiGradingMode}
+            groupWork={assessment.group_work}
+            rubricData={rubric_data}
+            instanceQuestionGroups={instanceQuestionGroups}
+            courseStaff={courseStaff}
```

Comment:
This is a raw `User` object. This should very much be using `StaffUserSchema`.

All of these mistakes are making me think that we really really really need better tooling to catch things like this.